sanic/sanic.py

```diff
@@ -20,7 +20,7 @@
 
 class Sanic:
     def __init__(self, name=None, router=None,
-                 error_handler=None, logger=None):
+                 error_handler=None, logger=None, backlog=100):
```

Comment:
I think that it is better that `backlog` is an argument of `sanic.run()`,
like `host` and `port`.